core/Inst.hpp

```diff
@@ -135,13 +137,26 @@ namespace olympia
         std::string getDisasm()   const { return opcode_info_->dasmString(); }
         uint32_t    getOpCode()   const { return static_cast<uint32_t>(opcode_info_->getOpcode()); }
 
+        // Operand information
+        using OpInfoList = mavis::DecodedInstructionInfo::OpInfoList;
+        const OpInfoList& getSourceOpInfoList() const { return opcode_info_->getSourceOpInfoList(); }
+        const OpInfoList& getDestOpInfoList()   const { return opcode_info_->getDestOpInfoList(); }
+
         // Static instruction information
         bool        isStoreInst() const    { return inst_arch_info_->isLoadStore(); }
         uint32_t    getExecuteTime() const { return inst_arch_info_->getExecutionTime(); }
 
         uint64_t    getRAdr() const        { return target_vaddr_ | 0x8000000; } // faked
         bool        isSpeculative() const  { return is_speculative_; }
 
+        // Rename information
+        RegisterBitMask & getSrcRegisterBitMask(const RegFile rf) {
+            return src_reg_bit_masks_[rf];
+        }
+        RegisterBitMask & getDestRegisterBitMask(const RegFile rf) {
+            return dest_reg_bit_masks_[rf];
+        }
```

Comment:
Probably want `const` version of these routines too.

Reply:
Done.